### PR TITLE
Refactoring styles

### PR DESCRIPTION
**Pull request by rvsia**
**Description**

- fixing tabs, toolbar styles
- fixing main title
- fixing some errors (proptypes and props)
- removing separate toolbar components (both toolbar components actually used the same one.. so, there is no need to have them)
- makes delete button red

**Before**
Screenshot from 2019-04-16 10-32-46
Screenshot from 2019-04-16 10-32-36

**After**
Screenshot from 2019-04-16 10-31-34
Screenshot from 2019-04-16 10-31-16

### CODE REVIEW

**Comment by lgalis** [comment]
Please merge only after https://github.com/ManageIQ/approval-ui/pull/10

**Comment by rvsia** [comment]
@Hyperkid123

File: src/App.js [hunk 1]
```diff
@@ -32,15 +32,14 @@ class App extends Component {
...
             Approval
           </Title>
+          <AppTabs />
         </PageHeader>
-        <Main style={ { backgroundColor: 'd3d3d3', margin: '0', padding: '0' } }>
-          <AppTabs>
-            <Routes childProps={ this.props } />
-          </AppTabs>
+        <Main>
+          <Routes childProps={ this.props } />
```
Comment (Hyperkid123) [inline src/App.js]: You can remove `childProps`.

File: src/smart-components/request/request-list.js [hunk 2]
```diff
@@ -33,16 +31,12 @@ class RequestList extends Component {
 
     return (
       <React.Fragment>
-        <div>
-          { this.props.isLoading && (<span color={ '#00b9e4' }> Loading...</span>) }
-        </div>
         { (this.props.items && this.props.items.length > 0) && (
-          <DataList aria-label="Expandable data list">
+          <DataList aria-label="Expandable data list" >
```
Comment (Hyperkid123) [inline src/smart-components/request/request-list.js]: extra space

**Pull request merged**